news/2020-04-07-mappers-report-march.md

```diff
@@ -0,0 +1,156 @@
+
+For osu!catch:
+- **[Dashes and Higher-Snapped Dashes on Rain difficulties](https://osu.ppy.sh/community/forums/topics/1033696)**
+
+For osu!mania:
+- **[Implementing Key-Mode Specific Ranking Criterias for every Key-Mode](https://osu.ppy.sh/community/forums/topics/1009849)**
+
+These and many others are currently discussed and need more input. So head right over to the Ranking Criteria Forum and be part of the changes which will affect the future of mapping!
+
+## <a id=”nat-meetings”></a>NAT Meetings
+
+As mentioned in the past issue of the Mappers’ Report we were holding regular meetings. This month was no exception of it. With these two meetings we also managed to go through all pending discussion points we had, and meetings will not be held less regular, means only when we have something important to discuss.
+
+If you have any topic which should be discussed in future meetings, please do not hesitate to contact any NAT member about it. If it’s about smaller things you can reach out to them right away. If it’s about something bigger, please consider to make a proposal first and post this one to a NAT member. 
+
+Once enough topics are gathered, future NAT meeting dates will be announced in further Mappers’ Report issues, in #modding on the [osu!dev Discord server](https://discord.gg/ppy) and in the forum!
+
+### [Mar 1st, 2020 - Cultivating BNs](https://osu.ppy.sh/community/forums/topics/1040487)
+
+In the Cultivating BNs meeting, we discussed the State of BN subdivisions, Mentoring new BNs, as well as the current perceived lack of modding among BNs and whether or not that’s true. The main conclusion after reviewing these factors is that, at present, there are no plans to make changes in these areas - but it is good to make sure that’s the case! Several community members are also working on proposals and research that will be brought to the table in the near future as well.
+
+### [Mar 22nd, 2020 - State of QAH + misc](https://osu.ppy.sh/community/forums/topics/1040456)
+
+This meeting discussed the current state of the Quality Assurance Helpers (QAH) and Ranking Criteria subforum engagement as its main topics. 
+For the QAH topic, we revisited the current state of the QAH and focused on evaluating the new changes made since the previous meeting about this topic in early December.  The meeting helped to gather information about what features the BN website is currently missing that could help make QAH work better. Additionally, the NAT has decided that QAH activity will count as 1/4th of a nomination for BN activity, although this number may be adjusted in the future.
+For the RC subforum topic, the meeting reviewed the current status and hashed out a plan going forward. Whenever possible, Ranking Criteria topics will be mentioned in both the osu!dev modding channel and this Mappers’ Report you’re reading to help give proposals more visibility. If you didn’t see it yet, you can find the proposals in the section above!
```

Comment:
```suggestion
This meeting discussed the current state of the Quality Assurance Helpers (QAH) and Ranking Criteria subforum engagement as its main topics. 

For the QAH topic, we revisited the current state of the QAH and focused on evaluating the new changes made since the previous meeting about this topic in early December.  The meeting helped to gather information about what features the BN website is currently missing that could help make QAH work better. Additionally, the NAT has decided that QAH activity will count as 1/4th of a nomination for BN activity, although this number may be adjusted in the future.

For the RC subforum topic, the meeting reviewed the current status and hashed out a plan going forward. Whenever possible, Ranking Criteria topics will be mentioned in both the osu!dev modding channel and this Mappers’ Report you’re reading to help give proposals more visibility. If you didn’t see it yet, you can find the proposals in the section above!
```

forgot to double newline, needed so that it doesn't turn into one blob paragraph.